keyboards/geistmaschine/geist/info.json

```diff
@@ -0,0 +1,273 @@
+{
+    "manufacturer": "Geistmaschine",
+    "keyboard_name": "Geist",
+    "maintainer": "ebastler",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "ROW2COL",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true,
+        "encoder": true
+    },
+    "matrix_pins": {
+        "cols": [ "D2", "D5", "D4", "D7", "B6", "C7", "F7", "F6" ],
+        "rows": [ "D6", "B4", "B5", "C6", "F0", "D3", "F5", "F4", "F1", "E6" ]
+    },
+    "processor": "atmega32u4",
+    "url": "https://geistmaschine.io/",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0001",
+        "vid": "0x676D"
+    },
+    "encoder": {
+        "rotary": [
+            { 
+                "pin_a": "B0", 
+                "pin_b": "B7",
+                "resolution": 2
+            }
+        ]
+    },
```

Comment:
```suggestion
    },
    "community_layouts": ["65_ansi", "65_iso"],
```

Reply:
Thanks, noticed the missing headers from the Lint errors and patched already.